ui/apps/platform/cypress.config.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
             framework: 'create-react-app',
             bundler: 'webpack',
         },
-        viewportHeight: 400,
```

Comment:
I received the following error in the tests, so I'm upping the viewport height to accommodate. 

<img width="1552" alt="Screenshot 2024-05-17 at 3 09 15 PM" src="https://github.com/stackrox/stackrox/assets/4805485/bbb8190a-2f36-4ce7-88b8-868cd62b0f78">

I wanted to see if I could append the menu to `document.body` like we do in other cases, but it didn't seem like there was a straightforward way to do it.